### ISSUE TITLE
After closing settings menu, keyboard focus is lost

### ISSUE THREAD

**Issue by Poobslag** [Poobslag/turbofat#304]
When closing the settings menu, keyboard focus is lost. The user cannot use the keyboard to navigate the menu.
